### PR TITLE
[MNOE-107] Fix admin finance page

### PR DESCRIPTION
**Pull request by ouranos**
Tenant#current_billing_amount to be calculated via mnohub for performance reason (and also to
perform the currency conversion)

Admin Finance page:
- Fix invoices table amount
- Add extra finance KPIs

### CODE REVIEW

File: api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb [hunk 1]
```diff
@@ -13,11 +13,18 @@ def show
 
     # GET /mnoe/jpi/v1/admin/invoices/current_billing_amount
     def current_billing_amount
-      data = Rails.cache.fetch('tenant_admin/current_billing_amount', expires_in: ADMIN_CACHE_DURATION) do
-        billing = MnoEnterprise::Organization.all.map(&:current_billing).sum(Money.new(0))
-        {current_billing_amount: {amount: billing.amount, currency: billing.currency_as_string}}
-      end
-      render json: data
+      tenant = MnoEnterprise::Tenant.get('tenant')
```
Comment (BrunoChauvet) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb]: To get a singular resource, do we need to specify a fake id? Could it be `MnoEnterprise::Tenant.get` ?

File: api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb [hunk 2]
```diff
@@ -13,11 +13,18 @@ def show
...
+
+      # Backward compatibility with old MnoHub -> to remove later
```
Comment (BrunoChauvet) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb]: Probably add the tag `TODO` here

**Comment by BrunoChauvet** [comment]
I would just add a `TODO` comment with the required version of mnohub so it's easier to do some clean up (don't forget to add the same todo to the old_tenant factory)

File: api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb (hunk 1 above)
Comment (ouranos) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb]: It's not a fake id actually. It's the resource name:
```
>> MnoEnterprise::Tenant.get('tenant')
=> GET /api/mnoe/v1/tenant
```

I'll check if I can find something cleaner

Comment (ouranos) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb]: @BrunoChauvet @x4d3 
How about:
```ruby
module MnoEnterprise
  class Tenant < BaseResource
    def self.show
      self.get('tenant')
    end
  end
end
```
So we can use `MnoEnterprise::Tenant.show` in the controller? Or I could override `get`?

Comment (BrunoChauvet) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/invoices_controller.rb]: That looks cleaner indeed :+1:

**Comment by ouranos** [comment]
@BrunoChauvet see last commit

**Comment by BrunoChauvet** [comment]
Looks good

**Comment by ouranos** [comment]
Squashed

**Pull request merged**